src/ecrc-frontend/src/components/page/orgValidation/OrgValidation.js

```diff
@@ -10,21 +10,22 @@ import SideCards from "../../composite/sideCards/SideCards";
 
 export default function OrgValidation({ page: { header, setOrg } }) {
   const [orgTicketNumber, setOrgTicketNumber] = useState("");
-  const history = useHistory();
   const [orgError, setOrgError] = useState("");
+  const [toTransition, setToTransition] = useState(false);
+  const [toOrgVerification, setToOrgVerification] = useState(false);
 
   const orgValidation = () => {
     axios
       .get(`/ecrc/doAuthenticateUser?orgTicketId=${orgTicketNumber}`)
       .then(res => {
         setOrg(res.data.accessCodeResponse);
-        history.push("/ecrc/orgverification");
+        setToOrgVerification(true);
```

Comment:
actually, lets just do redirect here inside - will reduce amount of local state :)